gui/default/index.html

```diff
@@ -641,28 +641,22 @@ <h4 class="panel-title">
                       </td>
                     </tr>
                     <tr>
-                      <th><span class="fas fa-fw fa-sitemap"></span>&nbsp;<span translate>Listeners</span></th>
+                      <th><span class="fas fa-fw fa-sitemap"></span>&nbsp;<span translate>Active Listening Addresses</span></th>
```

Comment:
I just went with the suggestion from @calmh here.  Having two (some / total) counts justifies to label what the difference in numbers signifies IMO.

Reply:
That doesn't change much that I am not a fan of making it longer like this. I am also not dead set on not having it. Just dreading e.g. the desig bug report about ultra-long labels not wrapping nicely (probably from a German for "Aktivierte zuhörende Addressen") :) 

As to having two counts and active: I could understand "active" in this context to mean active / available, while it is working / active. In my opinion it's the job of the modal to make clear what those two numbers mean.

Reply:
> Just dreading e.g. the desig bug report about ultra-long labels not wrapping nicely (probably from a German for "Aktivierte zuhörende Addressen") :)

Long labels should wrap fine in the current state, but they don't look very nice on small screens. I would agree that 1-2 words maximum would indeed be more preferable.

Reply:
I understood "active" to mean working, not configured.  Dunno about the translations, in the settings dialog it doesn't seem overly long.  Anyway, I was hoping for an opinion from @calmh, because he suggested these labels.

Reply:
I like the modal. I don't like the name change and am not interested in discussing it. If the name change is reverted, I am happy to do a final round of review and merge this. You can still bring up the name change in a separately.

Reply:
Sorry, I was under the impression that this was the consensus from the forum discussion.  You either didn's speak up loud enough there, or have a minority opinion ;-)